core/src/core.cpp

```diff
@@ -231,6 +268,11 @@ bool yukarin_s_forward(int64_t length, int64_t *phoneme_list, int64_t *speaker_i
     return false;
   }
   auto [model_index, model_speaker_id] = get_model_index_and_speaker_id(*speaker_id);
+  auto model = std::move(status->yukarin_s_list[model_index]);
```

Comment:
```suggestion
  auto &model = status->yukarin_s_list[model_index];
```
ここでmoveしてしまうとSessionが開放されてしまうのではないでしょうか？
参照にしたほうが良いと思います

Reply:
なるほどです･･･。仰る通りなので参照にしたいと思います。